freshmaker/handlers/koji/rebuild_images_on_rpm_advisory_change.py

```diff
@@ -319,6 +319,9 @@ def _record_batches(self, batches, event, builds=None):
                     for content_set in image["content_sets"]:
                         if content_set in image["compose_sources"]:
                             continue
+                        # Exclude hidden content set.
+                        if content_set.endswith(conf.hidden_content_set_pattern):
```

Comment:
Could you check with regex matching instead of `endswith`?

Reply:
Not sure about which is better between regex and endswith, my original code is using regex, Lukas commented better with endswith, both seems fine for me.

Reply:
I commented on `content_set.endswith("-hidden-rpms")` - so previously it looked like suffix-only, but it would be better with full regex match.

Reply:
Also, the config name is confusing because it contains `pattern`, but it is still simple string (and suffix) - not a regex nor glob expression. Either use `hidden_content_set_suffix` or use `hidden_content_set_pattern` but with regex matching.

Reply:
Ah,ok, got your point finally. Thanks for your explanation. Again, which one you prefer? endswith or regex, I use regex originally because I don't know the correct format for hidden set, I think string "hidden" must be there. So use "hidden" regex, I am assuming the "-hidden-rpms" should be more accurate after Lukas review, so I changed to endswith.